subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -3567,9 +3565,20 @@ isr_rx_conn_pkt(struct radio_pdu_node_rx *node_rx,
 					 * with MIC failure.
 					 * This could be an unencrypted packet
 					 */
-					pdu_data_rx = radio_pkt_scratch_get();
-					unknown_rsp = pdu_data_rx;
-					mic_failure = false;
+					struct pdu_data *scratch_pkt
+						= radio_pkt_scratch_get();
```

Comment:
minor: for consistency the line split should happen after the operator, i.e. `=` in this case, but in general I can see the pain with this function since you've got indentation 6 tabs deep :)

Reply:
Thank you, the indentation depth gave me some challenges :)